src/renderers/webgl/WebGLShadowMap.js

```diff
@@ -70,10 +71,12 @@ function WebGLShadowMap( _renderer, _lights, _objects, capabilities ) {
 
 		var useMorphing = ( i & _MorphingFlag ) !== 0;
 		var useSkinning = ( i & _SkinningFlag ) !== 0;
+		var useInstancing = ( i * _InstancingFlag ) !==0;
```

Comment:
> Can you see that anyone would want to use skinning and morphing with this?

Lets wouldn't support skinning and morphing for now... 😅

> Also, any thoughts on 
>  
>     var cluster = new THREE.InstancedMesh( geometry , material , 1000 , optionalPlacementFunction );

I would prefer keeping it simple:

```js
var cluster = new THREE.InstancedMesh( geometry, material, 1000 );
```